amavisvt/client.py

```diff
@@ -610,8 +606,8 @@ def check_vt(self, checksums):
             logger.exception("Error asking virustotal about files")
 
     def report_to_vt(self, resource):
-        if self.config.pretend:
-            logger.info("NOT sending resource to virustotal")
+        if self.config.pretend or self.config.do_not_report:
+            logger.info("NOT reporting resource to virustotal")
```

Comment:
Added new parameter do-not-report in config if it helps to save some calls to VT